src/components/Trade/hooks/useSwapper/calculateAmounts.test.ts

```diff
@@ -19,12 +18,11 @@ describe('calculateAmounts', () => {
       amount,
       buyAsset,
       sellAsset,
-      buyAssetUsdRate,
-      sellAssetUsdRate,
+      buyAssetFiatRate,
+      sellAssetFiatRate,
       action,
-      selectedCurrencyToUsdRate,
-      buyAssetTradeFeeUsd: bn(0), // A temporary shim so we don't propagate new tradeFee logic to V1 Swapper
-      sellAssetTradeFeeUsd: bn(0), // A temporary shim so we don't propagate new tradeFee logic to V1 Swapper
+      buyAssetTradeFeeFiat: bn(0), // A temporary shim so we don't propagate new tradeFee logic to V1 Swapper
+      sellAssetTradeFeeFiat: bn(0), // A temporary shim so we don't propagate new tradeFee logic to V1 Swapper
```

Comment:
Are these comments still valid, i.e are we going to remove these @0xApotheosis? Now that Swapper v1 is gone, these theoretically shouldn't be consumed anymore

Reply:
We ackchyually ended up using them in v2 swapper, so these comments are invalid - but I have a branch higher in the stack that removes `calculateAmounts` entirely, so it'll be cleaned up then 👌